assets/js/googlesitekit/notifications/components/common/Description.js

```diff
@@ -14,27 +14,37 @@
  * limitations under the License.
  */
 
+/**
+ * WordPress dependencies
+ */
+import { Fragment } from '@wordpress/element';
+
 /**
  * Internal dependencies
  */
 import { sanitizeHTML } from '../../../../util';
+import ErrorText from '../../../../components/ErrorText';
 
 export default function Description( {
 	className = 'googlesitekit-publisher-win__desc',
 	text,
 	learnMoreLink,
+	errorText,
 } ) {
 	return (
-		<div className={ className }>
-			<p>
-				<span
-					dangerouslySetInnerHTML={ sanitizeHTML( text, {
-						ALLOWED_TAGS: [ 'strong', 'em', 'br', 'a' ],
-						ALLOWED_ATTR: [ 'href' ],
-					} ) }
-				/>{ ' ' }
-				{ learnMoreLink }
-			</p>
-		</div>
+		<Fragment>
+			<div className={ className }>
+				<p>
+					<span
+						dangerouslySetInnerHTML={ sanitizeHTML( text, {
+							ALLOWED_TAGS: [ 'strong', 'em', 'br', 'a' ],
+							ALLOWED_ATTR: [ 'href' ],
+						} ) }
+					/>{ ' ' }
+					{ learnMoreLink }
+				</p>
+			</div>
+			{ errorText && <ErrorText message={ errorText } /> }
```

Comment:
Feels like this component is starting to do a bit too much maybe.

Reply:
I did consider creating a separate component but didn't because the `<ErrorText>` component already encapsulates the rendering of the error component which we forward a very simple message to for now. And adding it to description adds `errorText` support for all layouts rather than adding a new component/prop to every layout that might require it.